tests/Acoustics.Test/AnalysisPrograms/Recognizers/AustBitternTests.cs

```diff
@@ -66,12 +66,13 @@ public void TestRecognizer()
 
             var onlyEvent = (CompositeEvent)events[0];
 
-            Assert.AreEqual(5.12, onlyEvent.EventStartSeconds);
-            Assert.AreEqual(12.26, onlyEvent.EventEndSeconds);
+            Assert.AreEqual(3, onlyEvent.ComponentCount);
+            Assert.AreEqual(7.136, onlyEvent.EventStartSeconds);
+            Assert.AreEqual(12.224, onlyEvent.EventEndSeconds);
             Assert.AreEqual(105, onlyEvent.LowFrequencyHertz);
             Assert.AreEqual(180, onlyEvent.HighFrequencyHertz);
-            Assert.AreEqual(21.7, onlyEvent.Score);
-            Assert.AreEqual(0.95, onlyEvent.ScoreNormalized);
+            Assert.AreEqual(24.076815193551525, onlyEvent.Score, 0.01);
+            Assert.AreEqual(0.476507384367336, onlyEvent.ScoreNormalized, 0.01);
```

Comment:
```suggestion
            Assert.AreEqual(24.076815193551525, onlyEvent.Score, TestHelper.AllowedDelta);
            Assert.AreEqual(0.476507384367336, onlyEvent.ScoreNormalized, TestHelper.AllowedDelta);
```

See https://github.com/QutEcoacoustics/audio-analysis/blob/ed5579f3db4feac5dfde18fd90d13615d6015154/tests/Acoustics.Test/TestHelpers/TestHelper.cs#L28-L33